docs/SUPPORTED_ACTUATORS.md

```
@@ -563,6 +564,86 @@ Uses same 32-bit command values as Keeson - see [Keeson commands](#commands-32-b
 
 ---
 
+## Nectar
+
+**Status:** ⚠️ Untested
+
+**Credit:** Protocol reverse-engineered by [MaximumWorf](https://github.com/MaximumWorf) - [homeassistant-nectar](https://github.com/MaximumWorf/homeassistant-nectar)
+
+### Known Models
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)